src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+  @Test
+  @DisplayName("size() 메소드를 활용해 Set의 크기를 확인한다.")
+  void size() {
+    assertThat(numbers.size()).isEqualTo(3);
+  }
+
+//  @Test
+//  @DisplayName("Set의 contains() 메소드를 활용해 1,2,3의 값이 존재하는지 확인한다.")
+//  void contains() {
+//    assertThat(numbers.contains(1)) .isTrue();
+//    assertThat(numbers.contains(2)) .isTrue();
+//    assertThat(numbers.contains(3)) .isTrue();
+//  }
+
+  @ParameterizedTest
+  @ValueSource(ints = {1, 2, 3})
+  void contains(int number) {
+    assertThat(numbers.contains(number)) .isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(input);
```

collection의 경우 위와 같이 구현도 가능함